sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Security/Ucfg/UniversalControlFlowGraphBuilder.cs

```diff
@@ -60,7 +60,7 @@ public UCFG Build(SyntaxNode syntaxNode, IMethodSymbol methodSymbol)
                 Location = GetLocation(syntaxNode),
             };
 
-            ucfg.BasicBlocks.AddRange(cfg.Blocks.Where(IsSupportedBlock).Select(CreateBasicBlock));
+            ucfg.BasicBlocks.AddRange(cfg.Blocks.Select(CreateBasicBlock));
```

Comment:
Adding all blocks now, including Exit; helps with void methods (implicit returns) and exception throws.